documentation/requirements.txt

```diff
@@ -1,10 +1,11 @@
 # Python dependencies to build the documentation
 
 # != 3.5.0: https://github.com/sphinx-doc/sphinx/issues/8880
-Sphinx ~= 3.1, != 3.5.0
+Sphinx != 3.5.0
 breathe
```

Comment:
It's probably a good idea to pin to `~= 4.0` just so we don't end up with immediately breakage in random PRs when sphinx 5 is released.